ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-nonprod.json

```diff
@@ -70,5 +70,26 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "manageDocuments",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-systemupdate",
+          "caseworker-civil-admin",
+          "caseworker-civil"
+        ],
+        "CRUD": "CRUD"
+      },
+      {
+        "UserRoles": [
+          "APP-SOL-SPEC-PROFILE",
```

Comment:
can citizen also view this field? ( i.e. unrepresented defendant for example or unrepresented claimant)

Reply:
No, it will be only managed by caseworker, so removed these changes